doc/api_assets/style.css

```diff
@@ -16,7 +16,9 @@ body {
 }
 
 pre, tt, code, .pre, span.type, a.type {
-  font-family: Monaco, Consolas, "Lucida Console", monospace;
+  font-size: 1.2em;
+  line-height: 1.3em;
```

Comment:
I think we're better off setting `line-height` further up the cascade, like on `#column1` or `html` (affects the sidebar), and then remove all the `line-height`rules from the stylesheet.